Jint/Runtime/Debugger/BreakLocation.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Jint.Runtime.Debugger
+{
+    /// <summary>
+    /// BreakLocation is a combination of an Esprima position (line and column) and a source (path or identifier of script).
+    /// Like Esprima, first column is 0 and first line is 1.
+    /// </summary>
+    public sealed record BreakLocation
```

Comment:
btw, if you want really make the code short, this should also work:

```csharp
public sealed record BreakLocation(string Source, int Line, int Column)
{
    public BreakLocation(int line, int column) : this(null, line, column)
    {
    }

    public BreakLocation(string source, Esprima.Position position) : this(source, position.Line, position.Column)
    {
    }
}
```
 These are of course personal preferences and not important for this PR, I've just really liked the expressiveness they have 🙂

Reply:
Not a fan (yet) of positional records, when the properties aren't self-evidently positional (emphasized here by the multiple constructors). But that's also just philosophy. May change it anyway. 😉